src/MyArrayList.java

```diff
@@ -0,0 +1,489 @@
+import java.util.*;
+
+public class MyArrayList implements Iterable<Object> {
+//public class MyArrayList implements MyList {
```

Comment:
Why did you remove `implements MyList` and added `implements Iterable<Object>`? There should be both. In Java you can implement multiple interfaces